core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ddl.scala

```diff
@@ -135,9 +138,34 @@ case class CreateExternalTable(
     }
 
     val tableManipulation = resolved.asInstanceOf[TableManipulation]
-    tableManipulation.createExternalTable(tableIdent.table, provider, userSpecifiedSchema, options)
+    tableManipulation.createExternalTable(sqlContext, tableIdent.table, userSpecifiedSchema, options)
+
+    DdlUtils.registerTable(sqlContext, provider, resolved, tableIdent.table, tableIdent.database, Some(userSpecifiedSchema), options)
```

Comment:
Can't `DdlUtils.registerTable` be used at `ImportTablesUsingWithOptions#run`  to simplify its code? I'd use it there if it is possible